docs/source/advanced/multi_gpu.rst

```diff
@@ -362,6 +362,9 @@ Distributed Data Parallel Spawn
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 `ddp_spawn` is exactly like `ddp` except that it uses .spawn to start the training processes.
 
+.. note::
```

Comment:
Let's also add a note that DDP doesn't work in notebook.

Reply:
@tchaton It's already mentioned
https://pytorch-lightning.readthedocs.io/en/stable/advanced/multi_gpu.html#jupyter-notebooks